index.d.ts

```diff
@@ -354,7 +354,15 @@ export declare interface TracerOptions {
   flushInterval?: number;
 
   /**
-   *  Number of spans before partially exporting a trace. This prevents keeping all the spans in memory for very large traces.
+   * Enables or disables large payloads being sent to APM.
+   * This prevents keeping all the spans in memory for very large traces.
+   * @default true
+   */
+  partialFlushEnabled?: boolean
```

Comment:
Why would this ever be disabled? The only reason I could see would be to explicitly try to overload the agent, but that doesn't seem like a valid use case.

Reply:
I see in the motivation a mention of system tests, but tests should test the behaviour of the library, the library shouldn't be implemented for tests.

Reply:
Yeah that's fair. I don't necessarily think this is needed either, but I put it up for review with the intention of consistency across tracers. If internal discussion between tracers leads to the conclusion that we don't need this change (or changes like this that, in general, conform with `system-tests`), I'm happy to close out this PR!